Docker_Container/dockerfile

```diff
@@ -0,0 +1,20 @@
+# pull official base image
+FROM node:13.12.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY package-lock.json ./
```

Comment:
The refactored project seems to be using `yarn` as the package manager, so you will have to update this file to copy the `yarn.lock` file instead as done here [Building Docker image for Node application using Yarn dependency](https://stackoverflow.com/a/46744735), once #24 is merged

Reply:
ok sir changing them right now